### PR TITLE
bring schema dumping back

### PR DESCRIPTION
**Pull request by grosser**
... broke that in https://github.com/zendesk/samson/pull/2487 🤦‍♂ 
changing a few old migrations so they produce the schema we have in production ... there seem to be some rails bugs that make the old migrations not produce what they used to do (bigint / datetime etc)

### Risks
 - Low: column types mismatch for old apps that never migrated

### CODE REVIEW

**Review by yizhang-zen — APPROVED** [review]
Set risk to medium or low?

**Comment by grosser** [comment]
set to low: changing old migrations only influences apps that never ran them ... and these are all 1+ years old

File: db/schema.rb [hunk 1]
```diff
@@ -154,7 +154,7 @@
     t.string "name", null: false
     t.text "comment"
     t.string "owners"
-    t.index ["name"], name: "index_environment_variable_groups_on_name", unique: true
+    t.index ["name"], name: "index_environment_variable_groups_on_name", unique: true, length: 191
```
Comment (uthark) [inline db/schema.rb]: where does this arbitrary limit come from?

Comment (grosser) [reply db/schema.rb]: bug in 9fc22d6254 which reformatted the schema

Comment (grosser) [reply db/schema.rb]: https://github.com/zendesk/samson/pull/3656 to clean things up more

**Pull request merged**